Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteProjectController.java

```diff
@@ -60,114 +58,132 @@ public IRetrieveProjectManager getProjectManager() {
 			model.addAttribute("project", project);
 			return "sites/browseNetworks";
 		}
-		
+
 		return "NoNetworks";
 	}
-	
+
 	/**
-	 * This method gives the visualization of the network with the given network id 
-	 * @param networkId 					network id of the network that has to be visualized
-	 * @param model							ModelMap object to map values to view
-	 * @param principal						current session user
-	 * @return								returns a string to access the visualize network page of the project external website
-	 * @throws QuadrigaStorageException		Database storage exception thrown
-	 * @throws JAXBException				JAXB exception while getting the JSON
+	 * This method gives the visualization of the network with the given network id
+	 * 
+	 * @param networkId
+	 *            network id of the network that has to be visualized
+	 * @param model
+	 *            ModelMap object to map values to view
+	 * @param principal
+	 *            current session user
+	 * @return returns a string to access the visualize network page of the project external website
+	 * @throws QuadrigaStorageException
+	 *             Database storage exception thrown
+	 * @throws JAXBException
+	 *             JAXB exception while getting the JSON
 	 */
 	@RequestMapping(value = "sites/{projectUnixName}/networks/{networkId}", method = RequestMethod.GET)
-	public String visualizeNetworks(@PathVariable("projectUnixName") String unixName, @PathVariable("networkId") String networkId, ModelMap model, Principal principal) throws QuadrigaStorageException, JAXBException {
+	public String visualizeNetworks(@PathVariable("projectUnixName") String unixName, @PathVariable("networkId") String networkId, ModelMap model, Principal principal)
+			throws QuadrigaStorageException, JAXBException {
 		INetwork network = networkmanager.getNetwork(networkId);
-		if(network==null){
+		if (network == null) {
 			return "auth/accessissue";
 		}
 		IProject project = getProjectDetails(unixName);
 		model.addAttribute("project", project);
-		
+
 		ITransformedNetwork transformedNetwork = transformationManager.getTransformedNetwork(networkId);
-		
-		String nwId = "\""+networkId+"\"";
-		model.addAttribute("networkid",nwId);
+
+		String nwId = "\"" + networkId + "\"";
+		model.addAttribute("networkid", nwId);
 		String json = null;
-		if(transformedNetwork!=null){
+		if (transformedNetwork != null) {
 			json = d3Creator.getD3JSON(transformedNetwork.getNodes(), transformedNetwork.getLinks());
 		}
-		model.addAttribute("jsonstring",json);
+		model.addAttribute("jsonstring", json);
 		return "sites/networks/visualize";
 	}
-}
\ No newline at end of file
+
+	@RequestMapping(value = "sites/{ProjectUnixName}/about", method = RequestMethod.GET)
+	public String showAbout(@PathVariable("ProjectUnixName") String unixName, Model model, Principal principal) throws QuadrigaStorageException {
+		IProject project = getProjectDetails(unixName);
+		String projectid = project.getProjectId();

```

Comment:
if there is no project with that unix id, this will throw a null pointer exception